composer.json

```diff
@@ -37,7 +37,7 @@
         }
     },
     "require":           {
-        "php":                               "^5.6 || ^7.0",
+        "php":                               "^7.1",
         "doctrine/doctrine-module":          "^1.2",
         "doctrine/orm":                      ">=2.5,<2.7",
```

Comment:
[ORM's `2.6` version](https://github.com/doctrine/doctrine2/releases/tag/v2.6.0) is optimized to PHP 7.1 :+1:

Reply:
Yeah, this should be `^2.6.0`